### ISSUE TITLE
there should be an .nvda-addon link for users

### ISSUE THREAD

**Issue by signalsender103** [nvaccess/NVSpeechPlayer#1]
Since @nvaccess/audioscreen has a link to install a binary version of the add-on right on its README, I think it could be useful for NVSpeechPlayer to do the same. Thoughts?
